dev-lang/lua/lua-5.1.5-r107.ebuild

```diff
@@ -0,0 +1,153 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit multilib multilib-minimal portability toolchain-funcs
+
+DESCRIPTION="A powerful light-weight programming language designed for extending applications"
+HOMEPAGE="https://www.lua.org/"
+SRC_URI="https://www.lua.org/ftp/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="5.1"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~sparc64-solaris ~x64-solaris ~x86-solaris"
+IUSE="+deprecated readline"
+
+COMMON_DEPEND="
+	>=app-eselect/eselect-lua-3
+	readline? ( >=sys-libs/readline-6.2_p5-r1:0=[${MULTILIB_USEDEP}] )
+	!dev-lang/lua:0"
```

Comment:
```suggestion
	!dev-lang/lua:0"
# add a comment here why libtool needs to come from `CHOST`/SYSROOT and not `CBUILD`
```

Reply:
you're right; i added a comment summarizing the situation because it is surprising to someone building a sysroot